### PR TITLE
Add additional steps to SQL binding templates

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Also update the bundle version to v4 since we specify the v4 host

### CODE REVIEW

**Comment by Charles-Gagnon** [comment]
@dzsquared @chlafreniere @lucyzhang929 @MaddyDev

File: Functions.Templates/Templates/SqlInputBinding-Python/__init__.py [hunk 1]
```diff
@@ -6,11 +6,18 @@ def main(req: func.HttpRequest, items: func.SqlRowList) -> func.HttpResponse:
...
 
+    *IMPORTANT*
+        Local Development : You must have v4.x of the Azure Function Core Tools installed, support for earlier versions will be added in a later release.
+        Deployed App : The app must be deployed to the EUAP region, support for other regions will be added later.
+
+    See https://github.com/Azure/azure-functions-sql-extension/issues/250 for the current state of Python support for the SQL binding
+
     These tasks should be completed prior to running :
-    1. Update "commandText" in function.json - this should be the query to execute to retrieve the values being returned
+    1. Update "commandText" in function.json - this should be the name of the table that you wish to upsert values to
```
Comment (lucyzhang929) [inline Functions.Templates/Templates/SqlInputBinding-Python/__init__.py:16]: This line change is not needed right?

**Comment by Charles-Gagnon** [comment]
Decided to also update the bundle version to v4 since we specify the v4 host

**Review by dzsquared — APPROVED** [review]

**Review by chlafreniere — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
@soninaren This is ready for merging

**Pull request merged**